moecache.py

```diff
@@ -122,7 +134,7 @@ def connect(self):
             # thus, might read more than the current expected response
             # cleared on every reconnect since old bytes are part of old
             # session and can't be reused
-            self._buffer = ''
+            self._buffer = bytearray('', 'utf-8')
```

Comment:
Oh, buffer suppose to be modifiable.  But other literals should be just bytes.  Also drop the `'utf-8'` argument, not needed for non-Unicode literals.